test/utils.py

```diff
@@ -158,7 +158,7 @@ def test_should_not_validate_etag_when_set_as_false(self):
         self.mock_ov_client.connection.disable_etag_validation.assert_called_once()
 
 
-class ParamsTestCase(PreloadedMocksBaseTestCase):
+class FilterParamsTestCase(PreloadedMocksBaseTestCase):
```

Comment:
I prefer the name ParamsTestCase, since filter is only one of the four arguments you can provide: start, count, and filter.